arch/arm/core/aarch64/isr_wrapper.S

```diff
@@ -15,6 +15,7 @@
 #include <arch/cpu.h>
 #include <sw_isr_table.h>
 #include "macro_priv.inc"
+#include <drivers/interrupt_controller/gic.h>
```

Comment:
Leave locale includes as the last ones in the list

Reply:
> What's the problem you are seeing with the current implementation?

I have observed an instruction abort and determined it was due to reading 1023 from ICC_IAR and then trying to jump to the address at _sw_isr_table[1023] which is just some random unrelated data.  The exact cause of the spurious interrupt is not well understood yet, but my reading of GIC architecture (v2/v3/v4) indicates that it is both possible and legal to see this condition.  I looked at the Linux handling of this specific "special" interrupt ID, and they also just continue if this value is read.

It would also probably be a good idea to range check here against CONFIG_NUM_IRQS and cause some sort of predictable fatal error, but I didn't add that to this patch.

Reply:
> Leave local includes as the last ones in the list

Ack.